rasa_scheduler/celery_tasks.py

```diff
@@ -1,13 +1,16 @@
-import os
-
 import requests
 from celery import Celery
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'rasa_scheduler.settings')
+app = Celery('celery_tasks',
+             broker='redis://localhost:6379')
 
-app = Celery('rasa_scheduler')
-app.config_from_object('django.conf:settings', namespace='CELERY')
-app.autodiscover_tasks()
+app.conf.beat_schedule = {
+    'trigger-rasa-reminder': {
+        'task': 'celery_tasks.trigger_rasa_reminder',
+        'schedule': 2.0,
+        'args': (),
+    },
+}
```

Comment:
At some point this should maybe move to a separate file, but for now this helps for keeping the overview :)

Reply:
Agreed. But I would be in favour of having this all in one place (i.e. this file) until it really is not practical anymore.

Reply:
Check